templates/iyo/schema.capnp

```diff
@@ -0,0 +1,6 @@
+@0x9ed24f28a7f9b137;
+
+struct Schema {
+    # jwt token
```

Comment:
the schema is missing the client ID and client secret!
You assume that itsyouonline is already configured with name `main` while in reality u would like to configure the itsyouonline profile itself

Reply:
@muhamadazmy , I didn't add client ID and client secret because it gives complete access to itsyouonline. It's assumed that itsyouonline instance `main` is preconfigured.

Reply:
@FastGeert and @chrisvdg  could u give us your input on this ?

Reply:
The refreshable JWT approach is the way to go. @katia-e is correct that client-id / secret is not safe. So @katia-e check if you can use the itsyouonline client in Jumpscale to make this working. Otherwise check with reem how to make it possible, or maybe just refrain from using the itsyouonline client in jumpscale if it was not intended to be used as such, and just use the sal to refresh your JWT token as needed. We can add this maybe to the ovc client in jumpscale. I did the same thing for the zero-os client in Jumpscale.